build/scripts/build.ps1

```diff
@@ -410,6 +410,10 @@ function Test-XUnitCoreClr() {
             $args += " $dllPath"
             $args += " -xml " + (Join-Path $logDir ([IO.Path]::ChangeExtension($dllName, ".xml")))
 
+            # https://github.com/dotnet/roslyn/issues/25049
```

Comment:
>com [](start = 29, length = 3)

If there aren't too many tests that run into this problem, would it be better to keep the parallel runs, but skip those tests on CoreCLR?

Reply:
To clarify: our configuration files all disable parallelization in the test assemblies. Hence this isn't changing the behavior, it's just enforcing it directly until I can get configuration working again. Configuration is the better approach overall because it's more granular and can specify items other than parallelization (which we do in some cases)

---
In reply to: [171410538](https://github.com/dotnet/roslyn/pull/25050#discussion_r171410538) [](ancestors = 171410538)